src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -1,77 +1,88 @@
+        <b>Apply for the GI Bill and other education benefits</b>
+        <hr className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary" />
+        <p className="va-nav-linkslist-description">
+          Equal to VA Forms 22-1990 and 22-1995
+        </p>
+        <a
+          className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
+          href="/education/how-to-apply/"
+          onClick={onFeaturedContentClick(
+            'Apply for the GI Bill and other education benefits',
+          )}
+        >
+          <span>
+            Read more
+            <span className="vads-u-visibility--screen-reader">
+              about applying for the GI Bill and other education benefits
             </span>
-          </a>
-        </li>
-        <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
-          <b>Apply for VA health care benefits</b>
-          <hr className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary" />
-          <p className="va-nav-linkslist-description">
-            Equal to VA Form 10-10EZ
-          </p>
-          <a
-            className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
-            href="/health-care/apply/application/introduction"
-          >
-            <span>
-              Read more
-              <span className="vads-u-visibility--screen-reader">
-                about applying for VA health care benefits
-              </span>
-              <i className="fa fa-chevron-right vads-facility-hub-cta-arrow" />
+            <i className="fa fa-chevron-right vads-facility-hub-cta-arrow" />
+          </span>
+        </a>
+      </li>
+      <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
+        <b>Apply for VA health care benefits</b>
+        <hr className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary" />
+        <p className="va-nav-linkslist-description">Equal to VA Form 10-10EZ</p>
+        <a
+          className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
+          href="/health-care/apply/application/introduction"
+          onClick={onFeaturedContentClick('Apply for VA health care benefits')}
+        >
+          <span>
+            Read more
+            <span className="vads-u-visibility--screen-reader">
+              about applying for VA health care benefits
             </span>
-          </a>
-        </li>
-      </ul>
-    </>
-  );
-}
+            <i className="fa fa-chevron-right vads-facility-hub-cta-arrow" />
```

Comment:
Ah, caught us again 😧

Reply:
Just curious but why is this bot necessary? lol

Reply:
It's necessary because it wants to find our icons, and we have to stop it before it finds them all

Reply:
@bkjohnson can speak to this more, but I think essentially the purpose was to flag icons for accessibility review in case they're being used as triggers for user actions.

Reply:
@U-DON we're exploring ways to bypass the bot to add icons but so far I think we'd have to use  `dangerouslySetInnerHtml` but then of course the bot would flag that we disabled the linter, so we have to be a little clever.

```jsx
function ConcealedIcon({ className }) {
   // The bot looks for an <i> element, so we have to do it this way
    const i = document.createElement('i');
    i.className = className;
   
  // Dynamically construct the prop so we don't have to disable the linter
   const prop = ['dangerously', 'Set', 'Inner', 'Html'];
   const props = {
       [prop.join('')]: { __html: i.outerHtml },
   };
   
   return <span {...props} />
}
```

Reply:
🤣🙄

Reply:
I love how much effort you're putting into this @ncksllvn hahaha

Reply:
Yeah. We should talk about this during our team's next retro because the bot has caught our icons many times

Reply:
@ncksllvn and @kelsonic , I want to improve the messaging to make it more obvious what is happening and why. Feel free to leave feedback: https://github.com/department-of-veterans-affairs/vets-website/pull/12145

Reply:
Oh @bkjohnson I _think_ @ncksllvn was just joking 🙃But thank you for sharing!